pom.xml

```diff
@@ -44,9 +44,9 @@ under the License.
   <properties>
     <revision>2.0.1</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.266</jenkins.version>
+    <jenkins.version>2.277</jenkins.version>
     <java.level>8</java.level>
-    <jcasc.version>1.35</jcasc.version>
+    <jenkins-test-harness.version>1492.v843c23c9d568</jenkins-test-harness.version> <!-- TODO https://github.com/jenkinsci/jenkins-test-harness/pull/281 -->
```

Comment:
https://github.com/jenkinsci/jenkins-test-harness/pull/281

Reply:
Using `RealJenkinsRule` here because https://github.com/jenkinsci/saml-plugin/blob/de609d4a71113e603ec6e5b578024dbbc0638c87/src/main/java/org/jenkinsci/plugins/saml/OpenSAMLWrapper.java#L65 or https://github.com/jenkinsci/saml-plugin/blob/de609d4a71113e603ec6e5b578024dbbc0638c87/pom.xml#L96 make it less than obvious that plain `JenkinsRule` would simulate plugin operation as accurately as ATH.